sites/shopify.js

```diff
@@ -276,6 +274,7 @@ exports.guestCheckout = async ({
     taskLogger.info('Navigating to URL');
     await page.goto(url, { waitUntil: 'domcontentloaded' });
 
+    // perhaps the size id doesn't work with UK sizes?
```

Comment:
Is the bot erroring out here? Is this a TO-DO? Or what is the need for this comment?

Reply:
sorry thats a todo i will amend it

Reply:
@samc621 latest commit does not have todo